sw/device/tests/BUILD

```
@@ -390,10 +390,18 @@ opentitan_test(
 opentitan_test(
     name = "pwrmgr_random_sleep_all_wake_ups",
     srcs = ["pwrmgr_random_sleep_all_wake_ups.c"],
-    exec_env = {"//hw/top_earlgrey:sim_dv": None},
+    cw310 = new_cw310_params(
+        test_cmd= "--bootstrap={firmware}",
```

This probably needs a --bitstream to be programmed:

```suggestion
        test_cmd= " ".join([
            "--bootstrap=\"{firmware}\"",
            "--bitstream=\"{bitstream}\"",
        ]),
```

---

Done